### PR TITLE
add synonyms to MONDO:0016089 'infantile Krabbe disease'

### LINKED ISSUE #8029: Request for new synonyms: infantile Krabbe disease
### Mondo term

infantile Krabbe disease MONDO:0016089

### Synonyms

early onset Krabbe disease, early-onset Krabbe disease, early symptomatic Krabbe disease

### Synonym type

exact

### Additional comments

PMID:20301416
PMID:30777126

### PR DESCRIPTION
**Pull request by ehurwitz**
close #8029

### CODE REVIEW

**Comment by ehurwitz** [comment]
@sabrinatoro This is a PR adding the 3 exact synonyms (early onset Krabbe disease, early-onset Krabbe disease, early symptomatic Krabbe disease) for infantile Krabbe disease MONDO:0016089 requested.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -368936,6 +368936,9 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_subtype_of_a_disorder {source="Orphanet:206436"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "early onset Krabbe disease" EXACT []
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:368939]: Please add a "database cross-reference" annotation to notify where this synonym comes from (ie where is this name used). 
This database x-ref can be a PMID, another x-ref (eg Orphanet:206436), and/or an orcid)

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -368936,6 +368936,9 @@ subset: nord_rare {source="MONDO:NORD"}
...
+synonym: "early symptomatic Krabbe" EXACT []
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:368940]: Please add a "database cross-reference" annotation to notify where this synonym comes from (ie where is this name used). 
This database x-ref can be a PMID, another x-ref (eg Orphanet:206436), and/or an orcid)

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -368936,6 +368936,9 @@ subset: nord_rare {source="MONDO:NORD"}
...
+synonym: "early-onset Krabbe disease" EXACT []
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:368941]: Please add a "database cross-reference" annotation to notify where this synonym comes from (ie where is this name used). 
This database x-ref can be a PMID, another x-ref (eg Orphanet:206436), and/or an orcid)

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@ehurwitz Thank you for your PR. I am requesting some changes. Please let me know if you have questions. 
After you have addressed these changes, please assign back to me. Thank you.

**Comment by ehurwitz** [comment]
@sabrinatoro database cross-reference annotations (PMIDs) have been been added for each of the 3 synonyms. Please let me know if there are any other issues!

**Review by sabrinatoro — APPROVED** [review]

**Pull request merged**